### PR TITLE
Update Tox and Travis configs

### LINKED ISSUE #21: Update Tox and Travis configs.
There's a few errors running `tox` on a clean checkout (summary below).

I'm not quite sure what's wrong with `py27-dj18` and `py27-dj19` at this moment (will investigate) but `py32-dj19` and `py33-dj19` are no longer supported combinations. 

I will update the test matrix for both Tox and Travis to the supported Django/Python combinations. 

```
___________________________ summary _________________________________
  flake8-py27: commands succeeded
  flake8-py33: commands succeeded
  py26-dj14: commands succeeded
  py27-dj14: commands succeeded
  py26-dj15: commands succeeded
  py26-dj16: commands succeeded
  py27-dj15: commands succeeded
  py27-dj16: commands succeeded
  py32-dj15: commands succeeded
  py32-dj16: commands succeeded
  py33-dj15: commands succeeded
  py33-dj16: commands succeeded
  py27-dj17: commands succeeded
ERROR:   py27-dj18: could not install deps [flake8, coverage, django-discover-runner, https://github.com/django/django/archive/stable/1.8.x.zip#egg=django]; v = InvocationError('.../django-appconf/.tox/py27-dj18/bin/pip install flake8 coverage django-discover-runner https://github.com/django/django/archive/stable/1.8.x.zip#egg=django (see .../django-appconf/.tox/py27-dj18/log/py27-dj18-1.log)', 2)
ERROR:   py27-dj19: could not install deps [flake8, coverage, django-discover-runner, https://github.com/django/django/archive/master.zip#egg=django]; v = InvocationError('.../django-appconf/.tox/py27-dj19/bin/pip install flake8 coverage django-discover-runner https://github.com/django/django/archive/master.zip#egg=django (see .../django-appconf/.tox/py27-dj19/log/py27-dj19-1.log)', 2)
  py32-dj17: commands succeeded
  py32-dj18: commands succeeded
ERROR:   py32-dj19: commands failed
  py33-dj17: commands succeeded
  py33-dj18: commands succeeded
ERROR:   py33-dj19: commands failed
  py34-dj17: commands succeeded
  py34-dj18: commands succeeded
  py34-dj19: commands succeeded
```

### PR DESCRIPTION
**Pull request by carltongibson**
Fixes #21

### CODE REVIEW

**Comment by carltongibson** [comment]
@diox — This is ready to go. Are you happy for me just to push on or do you want to review? 

It doesn't look like Python 3.5 is ready on Travis yet. [See the build here](https://travis-ci.org/django-compressor/django-appconf/builds/80589964). I will investigate

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,27 @@
+sudo: false
 language: python
 env:
   - TOXENV=flake8-py27
   - TOXENV=flake8-py33
   - TOXENV=py26-dj14
   - TOXENV=py27-dj14
-  - TOXENV=py26-dj15
```
Comment (diox) [inline .travis.yml]: Do we need to remove those ? I know they are no longer supported versions of django, but I suspect some developers are still using them.

File: .travis.yml [hunk 2]
```diff
@@ -1,29 +1,27 @@
+sudo: false
```
Comment (diox) [inline .travis.yml]: :sunglasses:

**Comment by diox** [comment]
Looks good to me, merge when you think you are ready. Maybe we could add codecov here as well ?

File: .travis.yml (hunk 1 above)
Comment (carltongibson) [inline .travis.yml]: We don't **need to**... but for me it's a scope thing. We have limited time to support the project, and, despite the fact that **commercial** bodies use/rely on the project, nobody is going to jump in with an offer to support development — so we **do need** to draw the line tightly. IMO. Discuss. :-)

Comment (diox) [inline .travis.yml]: Fair enough, you convinced me :)

Comment (diox) [inline .travis.yml]: We should do the same for compressor then. It's actually a very good point.

**Comment by carltongibson** [comment]
> Maybe we could add codecov here as well ?

@treyhunner I'm guessing this is in your remit right? Could you do the same here?

File: .travis.yml (hunk 1 above)
Comment (carltongibson) [inline .travis.yml]: > We should do the same for compressor then

I'll open a PR.

**Pull request merged**